ext/gd/tests/imageflip.phpt

```diff
@@ -1,8 +1,10 @@
 --TEST--
 Testing imageflip() of GD library
+--EXTENSIONS--
+gd
 --SKIPIF--
 <?php
-if ( ! extension_loaded('gd') || !function_exists('imageflip')) die( 'skip GD not present; skipping test' );
+if (!function_exists('imageflip')) die( 'skip GD not present; skipping test' );
```

Comment:
Looks unnecessary, imageflip is not conditional.

Reply:
Oh, indeed. I fixed that, but generally the SKIPIF sections of the GD test suite could use some TLC.